jasmine/spec/inverted-index-test.js

```diff
@@ -1,10 +1,84 @@
-// asserting book.json is not empty
-var obj = new Index();
-obj.createIndex('../../books.json');
-var contents = obj.fileContents;
-
-describe("Read book data", function () {
-  it("Checks to assert that book.json is not empty", function () {
-    expect(contents.length).not.toBe(0);
+describe("Inverted Index Tests", function () {
```

Comment:
**Code Styling - Critical**

AirBnB style guide recommends use of single spaces all through. Refer to [this](https://github.com/airbnb/javascript#strings--quotes)